tools/topology/platform/intel/intel-generic-bt.m4

```diff
@@ -0,0 +1,78 @@
+#
+# BT audio offset support
+#
```

Comment:
Typo here, should be "offload".

Reply:
@yongzhi1 Btw, better to "open-code" BT to Bluetooth in all user-visible strings. In identifiers and filenames, "bt" is good (it's used widely in kernel bluetooth stack, there's e.g. CONFIG_BT), but without context, BT is not necessarily connected to Bluetooth.

Reply:
Ack.